### ISSUE TITLE
Composite boats cant switch ammo types

### ISSUE THREAD

**Issue by jal76** [spring1944/spring1944#334]
I tried everything, even in the script, it was easier before. (script is hard to follow)

**Comment by sanguinariojoe** [comment]
Sorry @jal76 , can you elaborate a bit more?

You mean that composite boats may use different ammo types but you cannot control that (with the classic button of prefer AP/prefer HE), right?

**Comment by sanguinariojoe** [comment]
Otherwise it is probably related with [this](https://github.com/spring1944/spring1944/issues/222)

**Comment by sanguinariojoe** [comment]
@jal76 ??
